From 7a1f745fd7a56a6d8ee6177ae6fa04f3e84fd1fb Mon Sep 17 00:00:00 2001
From: Dil3mm4 <dil3mm4.dev@gmail.com>
Date: Sat, 17 Nov 2018 18:18:42 +0000
Subject: [PATCH 3/4] Choose a more generic prop.

Since USSD it's a problem over MTK too and not only on Kirin, let's be more generic.

Change-Id: Icf1700f55be40915f9c64059019287f300d81405

Conflicts:
	src/java/com/android/internal/telephony/GsmCdmaPhone.java
---
 src/java/com/android/internal/telephony/GsmCdmaPhone.java | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/java/com/android/internal/telephony/GsmCdmaPhone.java b/src/java/com/android/internal/telephony/GsmCdmaPhone.java
index 1fa1621b6..7351257d4 100644
--- a/src/java/com/android/internal/telephony/GsmCdmaPhone.java
+++ b/src/java/com/android/internal/telephony/GsmCdmaPhone.java
@@ -2152,7 +2152,7 @@ public class GsmCdmaPhone extends Phone {
             // Complete pending USSD
 
             if (isUssdRelease) {
-                if (SystemProperties.getBoolean("ro.telephony.isHisiRIL", false)) {
+                if (SystemProperties.getBoolean("persist.sys.radio.ussd.fix", false)) {
                     found.onUssdFinished(ussdMessage, isUssdRequest);
                 } else {
                     found.onUssdRelease();
-- 
2.17.1

